packages/cli/cloud/src/deploy-project/action.ts

```diff
@@ -161,6 +162,15 @@ export default async (ctx: CLIContext) => {
   try {
     notificationService(`${apiConfig.apiBaseUrl}/notifications`, token, cliConfig);
     await buildLogsService(`${apiConfig.apiBaseUrl}/v1/logs/${buildId}`, token, cliConfig);
+
+    ctx.logger.log(
+      'To view the logs in the dashboard, please copy and paste the following URL into your web browser:'
+    );
+    ctx.logger.log(
+      chalk.blue.underline(
+        `${apiConfig.dashboardBaseUrl}/projects/${project.name}/deployments/${buildId}`
+      )
```

Comment:
not sure we want that blue underline, as some terminal already do their own thing with URLs 🤔

Reply:
My terminal doesn't at least!
And looking at "npm help npm", they do add underlines so I think the underline part is ok! The "blue" color though is debatable since it's not the most clear in a dark terminal :/

Reply:
let the underline maybe, and default color

Reply:
<img width="804" alt="Screenshot 2024-05-30 at 17 28 17" src="https://github.com/strapi/strapi/assets/55534657/48513f15-e2a2-4721-8b07-305620b70a7d">

Looks cleaner IMO!
Pushed the changes, doing a test round as a good measure